libbeat/docs/shared-systemd.asciidoc

```diff
@@ -0,0 +1,99 @@
+[[running-with-systemd]]
+=== { beatname_uc } and systemd 
+
+DEB and RPM packages include a service unit for Linux systems with systemd. On
+these systems { beatname_uc } can be managed with the usual systemd commands:
+
+==== Starting and stoping { beatname_uc}
+
+When using systemd, { beatname_uc } can be started or stopped with `systemctl`.
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl start {beatname_lc}
+------------------------------------------------
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl stop {beatname_lc}
+------------------------------------------------
+
+Service start on system boot can be enabled or disabled with `systemctl`,
+{ beatname_uc } is enabled when installed.
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl enable {beatname_lc}
+------------------------------------------------
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl disable {beatname_lc}
+------------------------------------------------
+
+
+==== { beatname_uc } status and logs
+
+Service status can be also obtained with `systemctl`.
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+systemctl status {beatname_lc}
+------------------------------------------------
+
+Logs are stored by default in journald, they can be viewed with `journalctl`.
+
+["source", "sh", subs="attributes"]
+------------------------------------------------
+journalctl -u {beatname_lc}.service
+------------------------------------------------
+
+NOTE: Unit file included in packages sets by default `-e` flag, that flag makes
+{ beatname_uc } to log to stderr and disables other log outputs. Systemd stores
+all output sent to stderr in journald.
+
+[float]
+=== Customizing systemd unit for { beatname_uc }
+
+Systemd service unit file includes some environment variables that can be
+overriden to change default options.
+[cols="<h,<,<m,<m",options="header",]
```

Comment:
Looks like your column attributes are off here. This is what I see when I build the docs:

![image](https://user-images.githubusercontent.com/14206422/49485632-7d3a8900-f7f0-11e8-81ba-d58e24df1309.png)